skype/skype-ps/skype/New-CsAutoAttendantCallableEntity.md

```diff
@@ -59,6 +60,14 @@ This example gets a user object using Get-CsOnlineUser cmdlet. We then use the A
 
 This example gets an application instance by name using Find-CsOnlineApplicationInstance cmdlet. We then use the AAD ObjectId of that application instance to create an application endpoint callable entity.
 
+### -------------------------- Example 5 --------------------------
+```powershell
+ $callableEntityId = (Find-CsGroup -SearchQuery "Main Auto Attendant") -MaxResults 1 -MailEnabledOnly $true | Select-Object -Property Id
+ $callableEntity = New-CsAutoAttendantCallableEntity -Identity $callableEntityId -Type SharedVoicemail -EnableTranscription
+```
+
+This example gets an mail-enabled group by name using Find-CsGroup cmdlet. We then use the ObjectId of that group to create a shared voicemail callable entity that supports transcription.
```

Comment:
I think we need to be specific and mention that it needs to be O365 group for shared voicemail and not a normal group. Maybe even have an example finding the O365 group

Reply:
I'm not sure I understand the comment
> example finding the O365 group

isn't `$callableEntityId = (Find-CsGroup -SearchQuery "Main Auto Attendant") -MaxResults 1 -MailEnabledOnly $true | Select-Object -Property Id` sufficient?

Reply:
There are different type of groups (security, distritbution, O365) you can create and find using Find-CsGroup. I just feel we need to specify that shared voicemail ONLY works with O365 groups

Reply:
How about after my latest commit?
I updated the docs to explicitly mention it has to be an O365 mail-enabled group.

`This example gets an O365 mail-enabled group by name using Find-CsGroup cmdlet. We then use the ObjectId of that group to create a shared voicemail callable entity that supports transcription.`